src/token-list.json

```diff
@@ -793,5 +793,25 @@
     "extensions": {
       "coingeckoId": "kineko"
     }
-  }
+  },
+   {
+      "chainId": 101,
+      "address": "FTkj421DxbS1wajE74J34BJ5a1o9ccA97PkK6mYq9hNQ",
+      "symbol": "MINECRAFT",
+      "name": "Synex Coin",
+      "decimals": 9,
+      "logoURI": "https://raw.githubusercontent.com/solana-labs/token-list/main/assets/mainnet/FTkj421DxbS1wajE74J34BJ5a1o9ccA97PkK6mYq9hNQ/logo.png",
+      "tags": [
+        "utility-token",
+        "community-token"
+      ],
+      "extensions": {
+        "coingeckoId": "synex-coin",
+        "coinmarketcap": "https://coinmarketcap.com/currencies/synex-coin/",
```

Comment:
The only thing we need here is `coingeckoId`. The rest can be deleted to reduce file size.